2.0.0: test suite fails with cppcheck 2.13.1 (unrecognized command line option: "--template")

```console
+ cd geany-plugins-2.0.0
+ /usr/bin/make -O -j48 V=1 VERBOSE=1 check
Making check in po
make[1]: Entering directory '/home/tkloczko/rpmbuild/BUILD/geany-plugins-2.0.0/po'
INTLTOOL_EXTRACT="/usr/bin/intltool-extract" XGETTEXT="/usr/bin/xgettext" srcdir=. /usr/bin/intltool-update --gettext-package geany-plugins --pot
make[1]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/geany-plugins-2.0.0/po'
make[1]: Entering directory '/home/tkloczko/rpmbuild/BUILD/geany-plugins-2.0.0/po'
rm -f missing notexist
srcdir=. /usr/bin/intltool-update -m
if [ -r missing -o -r notexist ]; then \
  exit 1; \
fi
make[1]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/geany-plugins-2.0.0/po'
Making check in utils
Making check in src
/usr/bin/make  check-local
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/geany-plugins-2.0.0/utils/src'
/usr/bin/cppcheck \
        -q --template gcc --error-exitcode=2 \
        --library=../../build/cppcheck-geany-plugins.cfg \
        -I/usr/include/geany \
          \
        .
cppcheck: error: unrecognized command line option: "--template".
make[3]: *** [Makefile:882: check-cppcheck] Error 1
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/geany-plugins-2.0.0/utils/src'
make[2]: *** [Makefile:751: check-am] Error 2
make[1]: *** [Makefile:568: check-recursive] Error 1
make: *** [Makefile:613: check-recursive] Error 1
```

--- a/build/cppcheck-geany-plugins.cfg
+++ b/build/cppcheck-geany-plugins.cfg
@@ -1,5 +1,11 @@
 <?xml version="1.0"?>
 <def format="2">
+  <!-- Geany-Plugins macros -->
+  <define name="PLUGIN" value="&quot;foo&quot;"/>
+  <!-- GLib macros -->
+  <define name="G_GSSIZE_FORMAT" value="&quot;li&quot;"/>
+  <define name="G_GINT64_MODIFIER" value="&quot;l&quot;"/>
+  <define name="G_DIR_SEPARATOR_S" value="&quot;/&quot;"/>
   <define name="G_DEFINE_TYPE(TN, t_n, T_P)" value=""/>
   <define name="G_DEFINE_BOXED_TYPE(TypeName,type_name,copy_func,free_func)" value=""/>
 </def>

--- a/build/cppcheck.mk
+++ b/build/cppcheck.mk
@@ -3,7 +3,7 @@ if HAVE_CPPCHECK
 
 check-cppcheck: $(srcdir)
 	$(CPPCHECK) \
-		-q --template gcc --error-exitcode=2 \
+		-q --template=gcc --error-exitcode=2 \
 		--library=$(top_srcdir)/build/cppcheck-geany-plugins.cfg \
 		-I$(GEANY_INCLUDEDIR)/geany \
 		$(AM_CPPCHECKFLAGS) $(CPPCHECKFLAGS) \

